kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/clients/KafkaClients.java

```diff
@@ -36,7 +39,10 @@ public static KafkaClient getKafkaClient() {
      * @return the kaf client
      */
     public static KafClient kaf() {
-        return new KafClient();
+        if (kafClient == null) {
```

Comment:
What's this part of the change for?

Reply:
Sorry, this was part of a change for kaf that should not be committed. I will revert it